test/basic.jl

```diff
@@ -12,9 +13,16 @@ for i in 1:episodes
 
     while !ViZDoom.is_episode_finished(game)
         state = ViZDoom.get_screen_buffer(game)
-        print(size(state))
+        screen_shape = ViZDoom.get_screen_width(game), ViZDoom.get_screen_height(game), ViZDoom.get_screen_channels(game)
+        print(size(state), screen_shape)
+        # example reshaping screen size
+        image_state = reshape(state, screen_shape)
         r = ViZDoom.make_action(game, rand(actions))
         println("Reward $r")
-        sleep(sleep_time)
```

Comment:
Without sleep I remember the game will run very quickly……

Reply:
Yes this is just for the runtests.jl though so faster is better